Deployment.md

```diff
@@ -20,7 +20,7 @@ as with multi-machine setups. Docker Swarm is part of Docker, so if you have Doc
 
 2. You can edit `configs/config.env` and `configs/config.yaml` to set default passwords for superadmin, minio and mongodb.
 
-3. Run `run-swarm.sh` to initialize the cluster.
+3. Run `build-swarm.sh` to initialize the cluster.
```

Comment:
```suggestion
2.5: Optionally: run `build-swarm.sh` to build the Docker images locally on your machine (eg. if you've made any changes).

3. Run `run-swarm.sh` to initialize the cluster.
```

Reply:
I separate out the build and run into separate scripts, I guess the build script should be listed as optional..